.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
 python:
   - "2.7"
-install:
-  - pip install flake8
-  - pip install -r requirements.txt
-before_script:
-  - flake8 --ignore=F401,E501 .
-script:
-  - python ./manage.py test
+
+install: pip install -r requirements.txt --quiet
+
+jobs:
+  include:
+    - script: pip install flake8 --quiet && flake8 .
```

Comment:
Please move `flake8` installation to `install` section. It is not an issue to have extra modules installed during tests run.